kernel-module/build-for-jetson-drive-igpu-native.sh

```diff
@@ -20,4 +20,22 @@
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 # DEALINGS IN THE SOFTWARE.
 
+KERNEL_SRC_DIR="/usr/src/nvidia/nvidia-oot/"
```

Comment:
This is not kernel src but nvdia-oot modules installation path.

Reply:
Let me update the variable.

Reply:
Updated variable name.